### ISSUE TITLE
Move gameplay mods to its corresponding section

### ISSUE THREAD

**Issue by Joehuu** [ppy/osu#2250]
Will do this issue, need feedback. This will help distinguish the mod better with the color section it is in.
![](https://puu.sh/zLi2y/5d0763b01c.png)
When no fail is enabled, they couldn't pass the map (with other mods) from what skill they had. No fail didn't help them or reduce the difficulty of the map, but helped them get at least a score.

![](https://puu.sh/zLiO3/863d89fe50.png)
Perfect doesn't increase the diff. That person could've had the same score without it. This mod just saves the hassle of pressing retry so many times as compared to sudden death, and also adds this extra mod to your score saying you used it.

First off, the special section should have mods that are unranked and mods that can't be in a difficulty section(i.e. Key Amount).
**Spun Out should be in Difficulty Reduction** b/c it helps you in some way.
**No Fail should be in Special** b/c no help is involved but you not failing.
**Sudden Death and Perfect should be in Special** b/c nothing is making it harder.

Next is multimod. Mods get placed together if they have a similarity in some way(i.e. Autoplay and Cinema.)
**Relax and Autopilot should be a multimod** b/c one is movement with no clicking and vise versa.

Read **bold** for tl;dr. Might miss some other mods if this gets approved. Will have to get used to it **if this happens.** And will involve changing colors in web.

**Comment by shortthu** [comment]
This thing need to be on the forum probably, we need more discussion
Anyway I disagree with something, SD and PF does make it difficult because it create a stress on player, challenge themself while NF make player feel more comfortable to enjoy the game.

**Comment by Joehuu** [comment]
I disagree with you because **the stress you say is the time spent repeating the map, not the map in general.** And NF doesn't make you any comfortable (**you are failing intentionally**). You **only** use this mod if you can't **survive** the map or are **practicing** in a way. Anyway these mods **assist** and not inc or dec the diff in any way. The section was called formerly called "assisted" but I changed it since not all mods assist in that section.

**Comment by shortthu** [comment]
Anyway I think you should post this to the forum, [Feature Request](https://osu.ppy.sh/forum/4) look good. The current grouping have been used for years and GitHub isn't a good place to discuss because there isn't very much people here.
Based on your osu! profile it seems like you didn't post anything to the forum, so good luck with your first post xd

**Comment by peppy** [comment]
We are not using the feature request forum for lazer discussions. this is the correct place.

**Comment by shortthu** [comment]
Just feel like it's a controversial stuff so we could use more opinion by posting in the forum.
But if not then alright let's just leave it here.

**Comment by Joehuu** [comment]
My first suggestion was implemented. Still want
- `Sudden Death` and `Perfect` to be in automation, since it is automated for you to lose after a certain criteria is met (missing)
- `Relax` and `Autopilot` to be a multimod. One "clicks" and One "moves".

`No Fail` can stay because there is no more suitable place for it (still want it to move, but can't)

Lastly cinema's behavior should be changed to have no hitsounds playing, and be moved to conversion. The gameplay is not shown anyway, so it should be on every game mode.
